src/applications/ivc-champva/10-10D/pages/ApplicantDependentStatus.jsx

```diff
@@ -3,28 +3,21 @@ import ApplicantRelationshipPage, {
   appRelBoilerplate,
 } from '../../shared/components/applicantLists/ApplicantRelationshipPage';
 
-const KEYNAME = 'applicantDependentStatus';
-const PRIMARY = 'status';
-const SECONDARY = 'otherStatus';
+const PROPERTY_NAMES = {
+  keyname: 'applicantDependentStatus',
+  primary: 'status',
+  secondary: '_unused',
+};
 
 function generateOptions({ data, pagePerItemIndex }) {
-  const {
-    currentListItem,
-    personTitle,
-    applicant,
-    useFirstPerson,
-    relative,
-    beingVerbPresent,
-    relativePossessive,
-  } = appRelBoilerplate({ data, pagePerItemIndex });
-
-  const customTitle = `${applicant}’s status`;
-
-  const relativeBeingVerb = `${relative} ${beingVerbPresent}`;
-
+  const bp = appRelBoilerplate({ data, pagePerItemIndex });
```

Comment:
Got rid of the unnecessary destructuring here - we only access a couple of these values in this function, and the rest just need to be passed on in the return statement.